gpkit/small_scripts.py

```diff
@@ -56,6 +56,8 @@ def unitstr(units, into="%s", options="~", dimless='-'):
     if units and not isinstance(units, Strings):
         try:
             rawstr = ("{:%s}" % options).format(units)
+            if str(units.units) == "count":
+                rawstr = "1.0 count"
```

Comment:
Hmm, it wasn't working in printing solution arrays without that; they kept printing just as dimensionless.